pkg/storage/segments/repairer.go

```diff
@@ -29,6 +30,8 @@ type Repairer struct {
 	signer     signing.Signer
 	identity   *identity.FullIdentity
 	timeout    time.Duration
+	inProgress map[storj.Path]bool
```

Comment:
Since we will have multiple servers this won't be sufficient.

Reply:
what do you think would be better to make sure that the same injured segment doesn't have a 
repair job picked up multiple times?

Reply:
Not quite sure how to do it easily. There are options of just letting it be repaired twice or before repairing verify that things haven't changed with the pointer. That of course still might race.

Reply:
Should we make a ticket for this and fix it in a different PR? Since it does not break the test